components/script/dom/eventdispatcher.rs

```diff
@@ -137,17 +148,18 @@ pub fn dispatch_event(target: &EventTarget, pseudo_target: Option<&EventTarget>,
     assert_eq!(event.phase(), EventPhase::None);
     assert!(event.GetCurrentTarget().is_none());
 
+    // 1. Set event's dispatch flag.
+    event.set_dispatching(true);
+
+    // 2. Initialize event's target attribute to target override, if it is given, and target
+    // otherwise.
     event.set_target(match pseudo_target {
         Some(pseudo_target) => pseudo_target,
         None => target.clone(),
     });
 
-    if event.stop_propagation() {
-        return !event.DefaultPrevented();
-    }
-
-    event.set_dispatching(true);
-
```

Comment:
I think the removal of event.stop_propagation() is a mistake since it will fail the assert! in dispatch_to_listeners() so I will re-instate it - but I'm not sure where in the step of events it belongs.

As for the set_dispatching, this is moved here https://github.com/servo/servo/pull/9409/files#diff-e70183cc533bf121ea8a9c077ea36d90R152 since it's the first step